ch.vorburger.osgi.gradle/src/test/java/ch/vorburger/osgi/gradle/internal/tests/BuildServiceTest.java

```diff
@@ -17,16 +17,17 @@
  */
 package ch.vorburger.osgi.gradle.internal.tests;
 
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.awaitility.Awaitility.await;
+import java.io.File;
+import java.util.concurrent.Future;
 
 import ch.vorburger.osgi.gradle.internal.BuildService;
-import ch.vorburger.osgi.gradle.internal.BuildServiceImpl;
 import ch.vorburger.osgi.gradle.internal.BuildServiceListener;
-import java.io.File;
-import java.util.concurrent.Future;
+import ch.vorburger.osgi.gradle.internal.GradleBuildService;
 import org.junit.Test;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
 /**
  * BuildService Test (non-OSGi).
  *
```

Comment:
any chance we could have test coverage for the MavenBuildService as well? :smiley_cat: